doc/docs.md

```diff
@@ -6835,6 +6835,7 @@ sizeof
 spawn
 static
 struct
+thread
```

Comment:
`thread` is not a keyword. It is a type name, similar to `map`, `array`, `chan` etc.

Reply:
Fixed